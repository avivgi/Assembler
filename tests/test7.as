.define sz = 2
K: .data 22
LIST: .data 6, 8, sz

; zero operands
rts

END: hlt