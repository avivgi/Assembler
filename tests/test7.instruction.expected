Address: 100	value: 0	 word: 00000000100100
Address: 101	value: 105	 word: 00000110100110
Address: 102	value: 2	 word: 00000000001000
Address: 103	value: 104	 word: 00000110100010
